src/Compilers/CSharp/Test/Emit/PDB/PDBTests.cs

```diff
@@ -12789,5 +12790,41 @@ public void SingleFileWithNoMethodBody()
 </symbols>
 ");
         }
+
+        [Fact]
+        public void CompilerInfo_WindowsPdb()
```

Comment:
"WindowsPdb": should we also verify what happens when using `DebugInformationFormat.PortablePdb` or `.Embedded`?

Reply:
I don't think that's necessary. This feature is specific to Windows PDBs, the code paths for Portable PDB is entirely different.